x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.test.tsx

```diff
@@ -460,4 +464,42 @@ describe('EventsViewer', () => {
       });
     });
   });
+
+  describe('exceptions modal', () => {
+    test('it renders exception modal if "exceptionsModal" callback exists', async () => {
+      const wrapper = mount(
+        <TestProviders>
+          <MockedProvider mocks={mockEventViewerResponse} addTypename={false}>
+            <EventsViewer
+              {...eventsViewerDefaultProps}
+              exceptionsModal={exceptionsModal}
+              graphEventId=""
+            />
+          </MockedProvider>
+        </TestProviders>
+      );
+
+      await waitFor(() => {
+        wrapper.update();
+
+        expect(wrapper.find(`[data-test-subj="mock-exceptions-modal"]`).exists()).toBeTruthy();
+      });
+    });
+
+    test('it does not render exception modal if "exceptionModal" callback does not exist', async () => {
```

Comment:
i think for the future we might wanna take a look at the structure of these files so we can lessen the number of mounts we're using, especially in cases where this test block is identical to others in this file save for the expect function. Could be a way to speed up the tests

Reply:
That's a great point - @patrykkopycinski is working on refactoring this code, may be interesting to look at patterns there https://github.com/elastic/kibana/pull/73228